stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -9,14 +9,21 @@ if [ -n "$PLATFORM" ]; then
     # Use specific platorm port map
     PLATFORM_ARGS="--env PLATFORM=$PLATFORM"
 elif [ -d "/etc/onl" ]; then
-    # Use ONLP to find platform and it's library
+    # Use ONLP to find platform and its library
     PLATFORM_ARGS=$(ls /lib/**/libonlp* | awk '{print "-v " $1 ":" $1 " " }')
     PLATFORM_ARGS="$PLATFORM_ARGS \
               -v /lib/platform-config:/lib/platform-config \
               -v /etc/onl:/etc/onl"
 fi
 
-CONFIG_DIR=${CONFIG_DIR:-/root}
+if [ -n "$FLAG_FILE" ]; then
+    FLAG_FILE_MOUNT="-v $FLAG_FILE:/etc/stratum/stratum.flags"
+fi
+
+if [ -n "$CHASSIS_CONFIG" ]; then
+    CHASSIS_CONFIG_MOUNT="-v $CHASSIS_CONFIG:/etc/stratum/$PLATFORM/chassis_config.pb.txt"
```

Comment:
If `$PLATFORM` is not initialized while calling `start-stratum-container.sh`, `CHASSIS_CONFIG_MOUNT` will mount the file to `/etc/stratum/chassis_config.pb.txt`. Since this isn't the default location for chassis_config, `stratum-entrypoint.sh` will continue using the older chassis_config from respective platform folder.

Reply:
All scripts now require the presence of `$PLATFORM` (or `/etc/onl/platform`)